### PR TITLE
Community Release fixes

### PR DESCRIPTION
**Pull request by mendesfabio**

### CODE REVIEW

File: src/components/common/global/CommonsHeader.vue [hunk 1]
```diff
@@ -1,14 +1,21 @@
 <template>
   <grid :gtc="titleOnly ? '1fr' : '120px 1fr 120px'" class="x-wrap">
     <div justify-self-start v-if="!titleOnly">
-      <p class="x-text-button inline" v-if="restart" @click="openModal('reset')">
-        Restart
+      <p class="x-text-button inline levels-button mh-0p5" @click="$refs.levels.show()">
+        Levels
       </p>
```
Comment (MerlinEgalite) [inline src/components/common/global/CommonsHeader.vue:6]: Maybe we can remove it when being on the results page?

**Comment by MerlinEgalite** [comment]
Screenshot 2021-01-12 at 20 45 21
Here the menu on the left disappears like for mobile, but the width is not so tiny. Maybe we can wait a bit more for the width to shrink to make the menu on the left disappear?

**Comment by MerlinEgalite** [comment]
Screenshot 2021-01-12 at 20 45 53
Cylinders are not totally empty or full, should I create an issue?

**Comment by mendesfabio** [comment]
Yeah, I'd create a new issue for cylinders. Thanks for the feedback, I've just fixed that. Now we don't have the Levels button on the Results page and I decreased the size required to show the mobile version.

**Pull request merged**